.ci/scripts/button-left-click.py

```diff
@@ -20,21 +20,22 @@ def button_left_click(device, start_x, start_y, offset_x=0, offset_y=0):
     device.emit(uinput.ABS_X, start_x, syn=False)
     device.emit(uinput.ABS_Y, start_y, syn=False)
     device.syn()
-    time.sleep(0.1)
+    time.sleep(0.2)
 
     # Press button.
     device.emit(uinput.BTN_LEFT, 1)
-    time.sleep(0.1)
+    time.sleep(0.2)
 
     # If offset is set, shift cursor offset pixels.
     if (offset_x > 0 or offset_y > 0):
         device.emit(uinput.ABS_X, start_x + offset_x, syn=False)
         device.emit(uinput.ABS_Y, start_y + offset_y, syn=False)
         device.syn()
-        time.sleep(0.1)
+        time.sleep(0.2)
```

Comment:
move this sleep out of the `if` statement
```suggestion
    time.sleep(0.2)
```

also it could be interesting to transform it in a parameter of the script: `--delay 0.2`

Reply:
Ok, I think I know what you want. You want a delay between the touch_down and the touch_up